### PR TITLE
[READY]Solid Gear Metal: Liquid Cardboard Deluxe

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Buffs the agent cardboard from the stealth implant, makes some adjustments and balances to the stealth suit.

## Why It's Good For The Game

Agent cardboard good. Stealth suit good (too good), and needs some balancing.

## Changelog
:cl:
balance: Moving in an agent cardboard box (stealth implant) no longer diminishes your invisibility.
balance: The stealth suit now has a slightly lower tick interval for adding to the user's invisibility.
balance: The stealth suit now has slightly better armor than a normal syndicate turtleneck.
balance: The stealth suit now malfunctions and reveals the user once bumped.
balance: The stealth suit now malfunctions and reveals the user once EMP'd.
balance: Using a stealth suit and a stealth implant is no longer gamery. If your box is revealed, your suit deactivates and you turn visible again.
balance: Stamina loss now reveals the stealth suit user.
fix: The stealth suit can no longer reach 0 alpha.
tweak: The stealth suit now uses animations instead of simply changing the alpha. Fancy!
tweak: The stealth suit is now an MK. III stealth suit. MI13 put a lot of effort into this model.
/:cl:

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -3338,53 +3322,34 @@
...
-#include "modular_skyrat\code\game\gamemodes\changeling\changeling.dm"
-#include "modular_skyrat\code\game\gamemodes\changeling\traitor_chan.dm"
-#include "modular_skyrat\code\game\gamemodes\cult\cult.dm"
-#include "modular_skyrat\code\game\gamemodes\devil\devil_game_mode.dm"
-#include "modular_skyrat\code\game\gamemodes\dynamic\dynamic_rulesets_midround.dm"
-#include "modular_skyrat\code\game\gamemodes\dynamic\dynamic_rulesets_roundstart.dm"
-#include "modular_skyrat\code\game\gamemodes\monkey\monkey.dm"
-#include "modular_skyrat\code\game\gamemodes\revolution\revolution.dm"
-#include "modular_skyrat\code\game\gamemodes\traitor\traitor.dm"
-#include "modular_skyrat\code\game\machinery\prisonlabor.dm"
```
Comment (jjpark-kb) [inline tgstation.dme]: Ill just comment it on this... Why are there so many removals?

Comment (BobJoga) [reply tgstation.dme]: ok this wasn't supposed to happen hold on

Comment (BobJoga) [reply tgstation.dme]: ok fixed i think

File: tgstation.dme [hunk 2]
```diff
@@ -3322,34 +3338,53 @@
...
 #include "modular_skyrat\code\game\objects\items.dm"
+#include "modular_skyrat\code\game\objects\effects\landmarks.dm"
 #include "modular_skyrat\code\game\objects\items\BSRPD.dm"
 #include "modular_skyrat\code\game\objects\items\cards_ids.dm"
 #include "modular_skyrat\code\game\objects\items\kitchen.dm"
+#include "modular_skyrat\code\game\objects\items\religion.dm"
 #include "modular_skyrat\code\game\objects\items\weaponry.dm"
 #include "modular_skyrat\code\game\objects\items\circuitboards\machine_circuitboards.dm"
 #include "modular_skyrat\code\game\objects\items\devices\radio\electropack.dm"
-#include "modular_skyrat\code\game\objects\items\implants\implant_stealth.dm"
```
Comment (jjpark-kb) [inline tgstation.dme]: Is this removed and put somewhere else or?

Comment (jjpark-kb) [reply tgstation.dme]: Github being weird still I guess

Comment (BobJoga) [reply tgstation.dme]: fixed fuck

**Pull request merged**